pyproject.toml

```diff
@@ -4,11 +4,25 @@ build-backend = "pseudo_builder"
 
 [tool.release-gitter]
 git-url = "https://github.com/JohnnyMorganz/StyLua"
-extract-files = ["stylua"]
+extract-all = true
 format = "stylua-{system}-{arch}.zip"
-exec = "chmod +x stylua"
+exec = """python -c "
+import os
+import stat
+import platform
+
+f = 'stylua'
+s = platform.system()
+
+if s in ('Linux', 'Darwin') and os.path.exists(f):
+    current_mode = os.stat(f).st_mode
+    os.chmod(f, current_mode | stat.S_IEXEC)
+" """
+
```

Comment:
Why do we need all this extraction change?

I'd imagine the only thing needed would be the new entry you've added to `map-arch`

Reply:
Makes sense, thanks for clarifying